Os/MicroFs/MicroFs.hpp

```diff
@@ -99,8 +99,10 @@ struct MicroFsBin {
 
 struct MicroFsConfig {
     FwNativeUIntType numBins;           //!< The number of bins configured. Must be <= than MAX_MICROFS_BINS
+    FwNativeUIntType filler;            //!< filler bytes to ensure 8 byte alignment of this data structure
```

Comment:
There are several issues here:
1. This code seems to assume that sizeof(FwNativeUnitType) == 4. We can't assume that. FwNativeUintType does not have a defined architecture-independent size.
2. We should not be using FwNativeUintType anyway. It doesn't have a defined size.
3. A better way to ensure alignment in C++11 is to use alignas. If you can't use alignas for some reason, then you can write `U8 pad[4]` for example to add padding of 4 bytes.

Reply:
I'm going to merge it to my fork to get the other changes, then we'll deal with the alignment and native integer issues.